app/models/applicant.rb

```diff
@@ -1,4 +1,12 @@
 class Applicant < ApplicationRecord
+
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :description, presence: true
+
```

Comment:
validations look good, I looked into what we might need for validation when we add the status attribute to the application by the way. Looks like we can use inclusion: just like you have used presence: here and pass in any enumerable. So we could pass ["in progress", "submitted", "approved.....]